recipes/treeqmc/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "TREEQMC" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/molloy-lab/TREE-QMC/releases/download/2.0.0/TREE-QMC-2.0.0.zip
```

Comment:
Can we use the {{ name }} defined in line 1 for the software name in this URL?

Reply:
Sure.